inference-engine/samples/benchmark_app/inputs_filling.cpp

```diff
@@ -90,7 +89,12 @@ void fillBlobImage(Blob::Ptr& inputBlob, const std::vector<std::string>& filePat
                     size_t offset = imageId * numChannels * width * height + (((app_info.layout == "NCHW") || (app_info.layout == "CHW"))
                                                                                   ? (ch * width * height + h * width + w)
                                                                                   : (h * width * numChannels + w * numChannels + ch));
-                    inputBlobData[offset] = vreader.at(imageId).get()[h * width * numChannels + w * numChannels + ch];
+                    if (floatBlobImage) {
```

Comment:
Please use the template function instead of this

Reply:
Agree. The latest master uses template function.